### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by donaldli2020** [Prodigy-Hacking/ProdigyMathGameHacking#1322] [Questions] [answered]
Do I build at the shell terminal? It looks like it's working, but I'm not sure.

**Comment by PatheticMustan** [comment, marked as answer]
yes, for P-NP build with `tsc` or `npx tsc`.
for WCM build with `webpack`

**Reply by donaldli2020** [reply]
```
npm run build
```
seems to work for PHEx and WCM

**Comment by 00100000** [comment]
Did you make sure to install node.js and the necessary dependencies? This is all done through the command line, so yes, you should be running the build scripts from your shell.

**Reply by donaldli2020** [reply]
I did cd willsCheatMenu and npm i before building it

**Comment by Jane-we** [comment]
Why every time I press the toggle membership then when I get off and go back on again the membership is gone?

**Reply by afkvido** [reply]
It doesn't save.
